addons/web/static/src/views/form/button_box/button_box.xml

```diff
@@ -5,13 +5,12 @@
     <t t-set="allButtons" t-value="getButtons()" />
     <t t-set="visibleButtons" t-value="allButtons['visible']" />
     <t t-set="additionalButtons" t-value="allButtons['additional']" />
-    <div class="o-form-buttonbox oe_button_box" t-attf-class="{{ visibleButtons.length >= getMaxButtons() ? 'o_full' : 'o_not_full'}} {{this.props.class}}">
-        <t t-slot="{{ button_value }}" t-foreach="visibleButtons" t-as="button" t-key="button_value" />
-
+    <div class="o-form-buttonbox oe_button_box position-relative d-block text-end" t-attf-class="{{ visibleButtons.length >= getMaxButtons() ? 'o_full' : 'o_not_full'}} {{this.props.class}}">
```

Comment:
d-block already here, needed on o-form-buttonbox too? also check if really needed as div is supposed to be a block by default

Reply:
Indeed, it was not necessary. I removed it 👍